### DISCUSSION TITLE
FBAInbound BoxInput quantity

### DISCUSSION THREAD

**Discussion by NoxArt** [amzn/selling-partner-api-models#4383] [Q&A] [answered]
Hi, maybe it's a stupid question, but in BoxInput
https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference#boxinput

**quantity**
The number of containers where all other properties like weight or dimensions are identical.

I don't know what "containers" refer to ... we just ship e.g. 5 boxes in 1 shipment, each box contains a few dozens items of various sizes
For now I'm just hardcoding it to 1

Thank you

**Comment by mafge** [comment, marked as answer]
Hi @NoxArt,
thanks for reaching out!
This is used in case you ship multiple boxes with exactly the same content (and the boxes must have the same weight and dimensions). In that case you can just set `quantity` instead of copying all the information for each individual box. In case your boxes contain different items, setting `quantity` to 1 is the right way. Hope that helps!
Best,
Marc
Selling Partner Developer Services

**Reply by NoxArt** [reply]
I believe it helps, thank you very much
